### DISCUSSION TITLE
How to study?

### DISCUSSION THREAD

**Discussion by sts-007** [community/community#65041] [GitHub Education]
### Hi everyone,

I am beginners. How should I start study with github. Please can you tech me?Thank you.

**Comment by arafajs** [comment]
Some steps on how to start studying Git and GitHub:

- Create a GitHub account. This is free and can be done on the GitHub website.
- Install Git on your computer. This is a version control system that GitHub uses. You can find the installation instructions for your operating system on the Git website.
- Learn the basics of Git. There are many resources available online to help you learn the basics of Git. Here are a few:

1. Git Tutorial: https://git-scm.com/book/en/v2
2. Learn Git in 15 Minutes: https://try.github.io/
3. GitHub Guides: https://docs.github.com/en/get-started/

- Start using GitHub. The best way to learn GitHub is to start using it. Create a new repository, add some files, and commit your changes. You can also fork other people's repositories and make changes to them.
- Contribute to open source projects. A great way to learn Git and GitHub is to contribute to open source projects. There are many open source projects that are looking for contributors. You can find a list of open source projects on the GitHub website.

**Comment by sts-007** [comment]
Value thanks.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by arafajs** [reply]
@sts-007 You're Welcome. 💐
